app/views/SelectedUsersView/SelectedChipItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo } from 'react';
+
+import { ISelectedUser } from '../../reducers/selectedUsers';
+import Chip from '../../containers/Chip';
+
+const SelectedChipItem = memo(
+	({
+		item,
+		useRealName,
+		onPressItem
+	}: {
+		item: ISelectedUser;
+		useRealName: boolean;
+		onPressItem: (userItem: ISelectedUser) => void;
+	}) => {
+		const name = useRealName && item.fname ? item.fname : item.name;
+		const username = item.search ? (item.username as string) : item.name;
+
+		return <Chip text={name} avatar={username} onPress={() => onPressItem(item)} testID={`selected-user-${item.name}`} />;
+	},
+	(prevProps, nextProps) => {
+		if (prevProps.item?.name !== nextProps.item?.name) {
+			return false;
+		}
+		if (prevProps.item?.fname !== nextProps.item?.fname) {
+			return false;
+		}
+		if (prevProps.useRealName !== nextProps.useRealName) {
+			return false;
+		}
+		return true;
+	}
```

Comment:
We need to avoid this.
This is not reactive.
Why do we need to keep track of user name change?
If you just send primitive values as props, it's going to work out of box.